ci/docker/Dockerfile

```diff
@@ -16,7 +16,19 @@
 FROM apachegeode/geode-build
 ENTRYPOINT []
 
-# Setup Google APT repository
+RUN apt-get update
+RUN apt-get install -y apt-transport-https ca-certificates aptitude
+RUN apt-key adv --keyserver hkp://pgp.mit.edu --recv-keys 58118E89F3A912897C070ADBF76221572C52609D
+
+# Setup Docker
+ADD docker.list /etc/apt/sources.list.d/
+RUN apt-get update
+RUN apt-get purge lxc-docker
+RUN apt-get install -y golang cgroupfs-mount docker-engine
+
+# Setup utilities
+RUN apt-get install -y jq unzip vim htop
+# Setup Google APT repository and install packages
 RUN apt-get update && \
```

Comment:
Any reason not merge all the `apt-get update ... install` into one layer?

Reply:
Realistically no. Most of this is copypasta from other docker images. I'll do an optimization pass.